src/jit/hwintrinsiclistxarch.h

```diff
@@ -435,6 +434,34 @@ HARDWARE_INTRINSIC(PCLMULQDQ_IsSupported,                            "get_IsSupp
+// ************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
+//
+//  Software helper intrinsics
+//
+HARDWARE_INTRINSIC(SW_INTRINSIC_START,                               "SW_INTRINSIC_START",                               SSE,        -1,           0,            0,           {INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid},           HW_Category_SimpleSIMD,                        HW_Flag_NoFlag)
+
+//  SSE Intrinsics
+HARDWARE_INTRINSIC(SSE_SetAllVector128,                              "SetAllVector128",                                  SSE,        -1,           16,           1,           {INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid},           HW_Category_SimpleSIMD,                        HW_Flag_NoFlag)
+HARDWARE_INTRINSIC(SSE_SetVector128,                                 "SetVector128",                                     SSE,        -1,           16,           1,           {INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid},           HW_Category_SimpleSIMD,                        HW_Flag_NoFlag)
+
+
+//  SSE2 Intrinsics
+HARDWARE_INTRINSIC(SSE2_SetAllVector128,                             "SetAllVector128",                                  SSE2,       -1,           16,           1,           {INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid},           HW_Category_SimpleSIMD,                        HW_Flag_NoFlag)
+HARDWARE_INTRINSIC(SSE2_SetVector128,                                "SetVector128",                                     SSE2,       -1,           16,           1,           {INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid},           HW_Category_SimpleSIMD,                        HW_Flag_NoFlag)
+//
+//  End of software helper intrinsics
+//
+HARDWARE_INTRINSIC(SW_INTRINSIC_END,                                 "SW_INTRINSIC_END",                                 POPCNT,     -1,           0,            0,           {INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid,   INS_invalid},           HW_Category_SimpleSIMD,                        HW_Flag_NoFlag)
```

Comment:
I do not think another group in the table is necessary, please just giving them a flag.